packages/clay-color-picker/src/Hue.tsx

```diff
@@ -70,14 +93,18 @@ const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 				window.addEventListener('pointerup', removeListeners);
 			}}
 			ref={containerRef}
+			role="slider"
+			tabIndex={0}
 		>
 			<button
 				className="clay-color-pointer clay-color-range-pointer"
+				onBlur={() => {
+					selectorActive.current = false;
+				}}
 				style={{
 					background: `hsl(${value}, 100%, 50%)`,
 					left: x - 7,
 				}}
-				type="button"
```

Comment:
Can you test if adding this again causes the accessibility bug? because maybe if we remove this, pressing the button with the DatePicker inside the Forms will cause a submit. If this causes accessibility problems, I think we'll have to change the tag to just a `div`.